spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+describe PostsController, type: :controller do
+  render_views
+
+  describe 'GET #index' do
+    let(:user) { FactoryBot.create(:user) }
+
+    before do
+      get :index, params: { user_id: user.id }
+    end
+
+    it 'Should return successful response' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'Should render the index template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'Should include Default text on template' do
+      expect(response.body).to include('List of posts here')
+    end
+  end
+
+  describe 'GET #show' do
+    let(:post) { FactoryBot.create(:post) }
```

Comment:
* According to the requirement you are to Create a Request spec file for all your controllers, kindly move all your tests into a request spec file as stated in the requirement 
![image](https://github.com/anderCM/blog-app/assets/52497246/3205a009-2db9-4b24-a447-abb6dcd16cda)

Reply:
Thanks @Davidosky007